src/App.tsx

```diff
@@ -1,21 +1,38 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Home from './components/Home';
+import Auth from './components/Auth';
+import Registration from './components/Registration';
+import Search from './components/Search';
+import Result from './components/Result';
+import User from './components/User';
+import Pet from './components/Pet';
+import AddPet from './components/AddPet';
+import Comparison from './components/Comparison';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
         <a className="App-link" href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
           Learn React
         </a>
       </header>
+      <Router>
+        <Switch>
+          <Route exact path="/" component={Home}></Route>
```

Comment:
I would highly recommend to have all possible routes in a separate config. 

Create file under `src/shared/constants/routes.constants.ts`

```javascript
export const ROUTES = {
  root: '/',
  auth: '/auth',
  user: '/user'
  // etc...
}
```

and re-use it like

```JSX
//...
<Route exact path={ROUTES.auth} component={Auth}></Route>
//...
```